### ISSUE TITLE
Segmentation Fault

### ISSUE THREAD

**Issue by Aniq55** [Graph-Machine-Learning-Group/sgp#1]
Every ```tsl.datasets.prototype``` import results in Segmenration Fault.

**Comment by marshka** [comment]
I think there might be errors in your virtual environment, did you manage to solve the problem?

**Comment by MingxiLii** [comment]
same problem

**Comment by Aniq55** [comment]
@marshka No, I followed the instructions on the README, and also created a new virtual space altogether.
Can you try to set up the code in a new virtual environment and see if that works? Perhaps using Docker.

Thank you very much!

**Comment by NoHesitation1** [comment]
I'm having the same problem. Have you solved it?
